extensions/anonaddy/.eslintrc.json

```diff
@@ -5,6 +5,41 @@
     "node": true
```

Comment:
Could we use

```
{
  "root": true,
  "extends": ["@raycast"]
}
```

Reply:
Added!

Reply:
After adding this locally, I'm a bit confused. Does it mean to replace the rest configuration?